flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/utils/HiveFunction.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.utils;
+
+import org.apache.flink.table.types.DataType;
+
+/**
+ * This test class is for hive module HiveFunction.
+ */
+public interface HiveFunction {
```

Comment:
you can move all tests to hive module, we don't have to duplicate this class to blink planner module

Reply:
These test are for hacky. `HiveCatalogUseBlinkITCase` have covered these cases, we can just remove them.